app/views/directives/osc-key-values.html

```diff
@@ -53,6 +59,12 @@
           </a>
         </div>
 
+        <div ng-if="showCommmitWarning" class="has-error">
+          <span class="help-block">
+            Please add or clear this {{(keyTitle || 'key') | lowercase}}-{{(valueTitle || 'value') | lowercase}} pair
```

Comment:
We good with the blue link in the red text @rhamilto @sg00dwin ?

![screen shot 2016-05-26 at 12 44 00 pm](https://cloud.githubusercontent.com/assets/280512/15582567/a00b0690-233f-11e6-9fd8-acec0a5532c4.png)